resources/main.mjs

```diff
@@ -349,11 +349,19 @@ class MainBenchmarkClient {
 
     _setLocationHash(hash) {
         if (hash === "#home" || hash === "") {
-            window.location.hash = "#home";
+            if (window.location.hash !== hash)
+                window.location.hash = hash = "#home";
```

Comment:
Did you mean to include this change here?

Reply:
I can land that separately (fix for the navigation recursion that I accidentally landed previously)

Reply:
ACK, created separate #357